src/analyses/natural_loops.h

```diff
@@ -28,6 +28,79 @@ Author: Georg Weissenbacher, georg@weissenbacher.name
 
 #include "cfg_dominators.h"
 
+template <class, class>
+class natural_loops_templatet;
+
+/// A natural loop, specified as a set of instructions
+template <class P, class T>
+class natural_loop_templatet
+{
+  typedef natural_loops_templatet<P, T> natural_loopst;
+
+public:
+  typedef std::set<T> loop_instructionst;
+  loop_instructionst loop_instructions;
```

Comment:
Why is this public?

Reply:
Doh, fixed it now, and made these private as you say.